benchmark/sparse_blas/sparse_blas.cpp

```diff
@@ -127,9 +127,12 @@ void apply_sparse_blas(const char* operation_name,
                               allocator);
             auto gen_logger = create_operations_logger(
                 FLAGS_gpu_timer, FLAGS_nested_names, exec,
-                test_case[operation_name]["components"], allocator, 1);
+                test_case[operation_name]["components"], allocator,
+                repetitions);
             exec->add_logger(gen_logger);
-            op->run();
+            for (unsigned i = 0; i < repetitions; i++) {
+                op->run();
+            }
```

Comment:
should the repetitions be in detail?

Reply:
We do the same for preconditioners, and it's useful to have reliable timings for the different components in a runtime breakdown

Reply:
We do not do the same for solver. It will be slower. but I am fine with both.